OpenOversight/app/models/database.py

```diff
@@ -705,6 +705,28 @@ class Incident(BaseModel, TrackUpdates):
     )
 
 
+class Allegation(BaseModel, TrackUpdates):
```

Comment:
For some reason, I cannot get alembic to generate a migration for this table addition. Is there anything that looks out of place, @sea-kelp?

Reply:
I tried a number of things with the LPL OO alembic but couldn't get it to generate the revision. I was however able to cherry pick the commits (613489b9f14c9d8e8e79430ae5a04a939e060793..f2977e4575a3edd7663467ec9a692d5ccb4a8c2c) into our OpenOversight repo and generate the revision there. I'm wondering if there's maybe a change that we haven't ported over yet?
https://github.com/OrcaCollective/OpenOversight/commit/cadea7114e4c723d411dc3fbd7e91df708cc5652

Obviously, our models might be slightly different from yours so I wouldn't recommend directly using this generated migration

Reply:
I remember seeing some comments that each individual models needs to be imported into Alembic's `env.py` file. I'll take a look at the Orca one and see what the differences are.

Reply:
It's super hacky, but I added the table without it creating a migration and then removed the table and it created a migration. I'm just going to reverse the code on upgrade and downgrade, and take that result. I'm not super happy with how it ended up working, but I'll take it at this point.